### PR TITLE
Workaround core-js performance bug

### PR DESCRIPTION
**Pull request by davidmanzanares**
Core-js bug https://github.com/zloirock/core-js/issues/377 is making 10x slower CARTO VL on Angular applications.

Since Core-js is so common (https://www.npmjs.com/package/core-js https://github.com/zloirock/core-js/network/dependents) I think it's better to live with this workaround.

### CODE REVIEW

File: src/utils/util.js [hunk 1]
```diff
@@ -32,6 +32,15 @@ export function isObject (value) {
     return value !== null && (type === 'object' || type === 'function');
 }
 
+export function getFloat32ArrayFromArray (array) {
+    const length = array.length;
```
Comment (IagoLast) [inline src/utils/util.js]: Can we just? 

```js
getFloat32ArrayFromArray (array) {
  return new Float32Array(array);
}
```

Comment (IagoLast) [reply src/utils/util.js]: or in any case use [from](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/from) 

```js
getFloat32ArrayFromArray (array) {
  return new Float32Array.from(array)
```

Comment (davidmanzanares) [reply src/utils/util.js]: The first one, of course not, you are doing the same and you are hitting the same monkey patch, getting the same results.

Comment (davidmanzanares) [reply src/utils/util.js]: The second one though makes a lot of sense, and I actually tried it before, but no luck, the performance is bad too, core-js is probably monkey patching that too.

Comment (IagoLast) [reply src/utils/util.js]: 👍 good to know :)

**Pull request merged**